include/mc_state_observation/TiltObserver.h

```diff
@@ -0,0 +1,117 @@
+#pragma once
+
+#include <mc_observers/Observer.h>
+#include <state-observation/observer/tilt-estimator.hpp>
+#include <state-observation/tools/rigid-body-kinematics.hpp>
+
+namespace mc_state_observation
+{
+
+struct TiltObserver : public mc_observers::Observer
```

Comment:
Could you add some doc that clarify how this is different from the AttitudeObserver and when/why one might want to use it?